docs/reference/ilm/getting-started-ilm.asciidoc

```diff
@@ -1,15 +1,190 @@
 [role="xpack"]
+[testenv="basic"]
 [[getting-started-index-lifecycle-management]]
 == Getting started with {ilm}
 
-Create a policy that rolls over after 1 day deletes an index after 30 days
+Let's jump into {ILM} by working through a hands-on scenario.
+This section will leverage many new concepts unique to {ILM} that
+you may not be familiar with. The following sections will explore
+these in more details.
 
-Show create policy API req/res
+The goal of this example is to set up a set of indices that will encapsulate
+the logs from our logging source. We can imagine there is a system
+like {filebeat-ref}[Filebeat] that continuously indexes documents into
+our writing index. We wish to roll over the index after it reaches a size of 50 gigabytes,
+or has been created 30 days ago, and then delete the index after 90 days.
 
-Show assign policy to index API req/res
+=== Setting up a new policy
 
-Show both the API and how it is done with `index.lifecyce.name` using the
-create-index API
+There are many new APIs introduced by {ILM}, but we will only focus on
```

Comment:
```suggestion
There are many new features introduced by {ILM}, but we will only focus on
```

I don't feel strongly about this change but my motivation is that I don't think ILM introduces a lot of APIs.

Reply:
```suggestion
There are many features introduced by {ILM}, but we will only focus on
```

ILM will only be "new" for a little while